catalog/core/catalog-core-standardframework/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -364,6 +364,8 @@
     <reference id="defaultAttributeValueRegistry"
                interface="ddf.catalog.data.DefaultAttributeValueRegistry"/>
 
+    <reference id="attributeInjector" interface="ddf.catalog.data.AttributeInjector"/>
```

Comment:
Shouldn't this be a reference list?  That way integrators could provide their own implementations?